app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -224,14 +224,40 @@ private String getLastNavFragment() {
     private void checkFirstLaunch() {
         SharedPreferences prefs = getSharedPreferences(PREF_NAME, MODE_PRIVATE);
         if (prefs.getBoolean(PREF_IS_FIRST_LAUNCH, true)) {
-            new Handler().postDelayed(() -> drawerLayout.openDrawer(navDrawer), 1500);
-
             // for backward compatibility, we only change defaults for fresh installs
             UserPreferences.setUpdateInterval(12);
 
             SharedPreferences.Editor edit = prefs.edit();
             edit.putBoolean(PREF_IS_FIRST_LAUNCH, false);
             edit.commit();
+
+
+            if (UserPreferences.shouldShowOnboarding("MainActivity")) {
```

Comment:
maybe rather use the TAG
having a string in such a critical place just smells... can break just to easily and no tests will go red